master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml

```diff
@@ -102,9 +102,14 @@ spec:
         # if it ever gets evicted.
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
-      tolerations:
-      - key: CriticalAddonsOnly
-        operator: Exists
+       tolerations:
+        # Allow the pod to run on the master.  This is required for
+        # the master to communicate with pods.
+        - key: node-role.kubernetes.io/master
```

Comment:
Shouldn't this one be changed to `effect: NoSchedule` and `operator: exists` and the `key` dropped?

Reply:
Hm, I may have been overzealous with my replacing of tolerations, I'm not sure we actually need to update Typha tolerations, I think it just needs the `CriticalAddonsOnly`